src/test/javascript/spec/component/code-editor/code-editor-ace.component.spec.ts

```diff
@@ -31,10 +29,10 @@ describe('CodeEditorAceComponent', () => {
     let codeEditorRepositoryFileService: CodeEditorRepositoryFileService;
     let loadRepositoryFileStub: SinonStub;
 
-    beforeEach(async () => {
+    beforeEach(() => {
```

Comment:
question regarding the [`afterEach()`](https://github.com/ls1intum/Artemis/pull/3898/files#diff-e862740e6901bc51f75acbf53be5c124cd2815ca13425dc0d8452ad95e185720R52-R54), why we only need to restore the specific stub, we also use a spy, isn't better to change this to `sinon.restore()` so that we make sure that a newly introduced stub for example is not missing the `restore()` statement. And don't we need it anyways for the spy we are using?

Reply:
or do we rather explicitly restore single stubs? I saw that pattern is used in other test files within the changeset of this PR, so maybe we should leave it... Up to you.

Reply:
Imho we should use sinon.restore, since it is more resilient to someone forgetting to restore a stub. I don't think we have a usecase where you _don't_ want to restore a stub.

Reply:
I replaced the restore with a sinon.restore() :)